.github/workflows/tests.yml

```diff
@@ -28,60 +29,21 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          python-version: [3.6, 3.7, 3.8, 3.9]
-          requirements-level: [min, pypi]
-          db-service: [postgresql9, postgresql11, mysql5, mysql8]
-          search-service: [elasticsearch6, elasticsearch7]
-          exclude:
-          - python-version: 3.8
-            requirements-level: min
-
-          - python-version: 3.9
-            requirements-level: min
-
-          - db-service: postgresql11
-            python-version: 3.6
-
-          - db-service: mysql8
-            python-version: 3.6
-
-          - search-service: elasticsearch7
-            python-version: 3.6
-
-          - search-service: elasticsearch6
-            db-service: postgresql11
-
-          - search-service: elasticsearch6
-            db-service: mysql8
-
-          - search-service: elasticsearch7
-            db-service: postgresql9
-
-          - search-service: elasticsearch7
-            db-service: mysql5
+          python-version: [3.7, 3.8, 3.9]
+          requirements-level: [pypi]
+          db-service: [postgresql13]
```

Comment:
for this package I would leave previous postgres version since there are older instances still using it